tests/parsers/beckman_pharmspec/testdata/hiac_example_1.json

```diff
@@ -46,78 +46,78 @@
                                                     "distribution": [
                                                         {
                                                             "particle size": {
-                                                                "value": 2.0,
+                                                                "value": 2,
                                                                 "unit": "μm"
                                                             },
                                                             "cumulative count": {
-                                                                "value": 48.0,
+                                                                "value": 48,
                                                                 "unit": "(unitless)"
                                                             },
                                                             "cumulative particle density": {
-                                                                "value": 10.0,
+                                                                "value": 10,
                                                                 "unit": "Counts/mL"
                                                             },
-                                                            "distribution identifier": "BECKMAN_PHARMSPEC_TEST_ID_15"
```

Comment:
Quick question I understand that the ultimate effect of reordering the produced asm data may cause the dist identifiers to get shuffled but shouldnt the values end up shuffled too?

Similar to the change for ` "BECKMAN_PHARMSPEC_TEST_ID_7"` in line 356

Reply:
I don't think so. Will be redoing how these IDs are allocated, so we can revisit if it still looks wrong.

Reply:
@rayaankha-tara I posted an updated file for your review, if needed.